### PR TITLE
fixed jsp-archetype to allow versions

### PR DESCRIPTION
**Pull request by reinheimer1**
added jsf-archetype

### CODE REVIEW

File: hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/store/logs/logger.log [hunk 1]
```diff
@@ -0,0 +1,8013 @@
+2024-02-14 16:03:27 [deploy-portlet] [debug]: Deploy portlet execution started
```
Comment (ThomasHurek) [inline hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/store/logs/logger.log]: Do not think we want to check in the log file.

File: hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/src/main/java/pagecode/PageCodeBase.java [hunk 2]
```diff
@@ -0,0 +1,229 @@
+package pagecode;
```
Comment (ThomasHurek) [inline hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/src/main/java/pagecode/PageCodeBase.java:19]: Missing copyright in java files - see e.g. https://github.com/HCL-TECH-SOFTWARE/dx-portlet-development-utilities/blob/main/pageimport/src/main/java/com/hcl/dx/sample/pages/MyPortlet.java#L1

**Review by ThomasHurek — CHANGES_REQUESTED** [review]
See some comments.

File: hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/store/logs/logger.log (hunk 1 above)
Comment (reinheimer1) [reply hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/store/logs/logger.log]: I removed the log-file

File: hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/src/main/java/pagecode/PageCodeBase.java (hunk 2 above)
Comment (reinheimer1) [reply hcl_dx_jsf_demoportlet_archetype/src/main/resources/archetype-resources/src/main/java/pagecode/PageCodeBase.java:19]: I added the copyright to the java files

**Review by ThomasHurek — APPROVED** [review]

**Pull request merged**